src/Lean/Elab/Tactic/BuiltinTactic.lean

```diff
@@ -369,12 +369,36 @@ private def getCaseGoals (tag : TSyntax ``binderIdent) : TacticM (MVarId × List
   let gs ← getUnsolvedGoals
   let g ← if let `(binderIdent| $tag:ident) := tag then
     let tag := tag.getId
-    let some g ← findTag? gs tag | throwError "tag not found"
+    let some g ← findTag? gs tag | notFound tag gs
     pure g
   else
     getMainGoal
   return (g, gs.erase g)
 
+  where
```

Comment:
Style nit: we usually dedent the `where` block

Reply:
Thanks, I'll work on my Haskell accent :)

Reply:
Done